arcsi/api/show.py

```diff
@@ -389,29 +389,28 @@ def view_show_archive(show_slug):
 @auth_token_required
 def view_show_page(show_slug):
     do = DoArchive()
+    filterArray = {}
+    filterList = request.args.getlist('filter')
+    if len(filterList) == 1 and ',' in filterList[0]:
+        filterArray['filters'] = comma_separated_params_to_list(filterList[0])
+    else:
+        filterArray['filters'] = filterList 
+    # TODO workaround for current frontend usage
+    archived = 'archived' in filterArray['filters']
+    if archived == None:
+        if "https://lahmacun.hu" in request.environ.get('HTTP_ORIGIN'):
```

Comment:
🛖

Reply:
until this workaround is here, frontend can't use this query to get the future episodes (filters doesn't contain `archived`)